<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1332</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>621</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>mission</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>611</width>
        <height>331</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_id_miss">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>50</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_miss">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>110</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_lieu_miss">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>170</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>lieu</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>230</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_modif">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>40</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_modif_miss">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>100</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>lieu</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_lieu_modif_miss">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>170</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>240</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>supprimer</string>
       </attribute>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_supp_miss">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>200</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_miss">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>11</y>
          <width>401</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>30</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>afficher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_recherche_nom">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>130</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>70</y>
          <width>101</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>recherche nom</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>190</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_13">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>250</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>equipes </string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>621</width>
        <height>381</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_11">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>50</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_12">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>110</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_13">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>170</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>nombre</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>240</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
       <widget class="QComboBox" name="combo">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>220</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>230</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>id_mission</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_14">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>40</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_15">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>100</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>nombre</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_16">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>170</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>240</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>220</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>220</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>id_mission</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>supprimer</string>
       </attribute>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_17">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>200</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_4">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>11</y>
          <width>401</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_12">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>80</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>trier par nbr</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>412</x>
          <y>140</y>
          <width>141</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>statistique entre 1 et 5</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_stat1">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>170</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_14">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>210</y>
          <width>131</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>statistique entre 5et 10</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_stat2">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>260</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_15">
        <property name="geometry">
         <rect>
          <x>422</x>
          <y>270</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>statistique entre 11 et 15</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_stat3">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>310</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>370</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>destinataire</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="msg_4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>350</y>
      <width>256</width>
      <height>181</height>
     </rect>
    </property>
    <property name="plainText">
     <string>transfert ajouté avec succés!!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>110</y>
      <width>256</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>465</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>806</x>
      <y>160</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>nom_compte</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>serveur</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mot de passe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBtn_4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn_4">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subject_4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>310</y>
      <width>256</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>notification</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="server_4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>70</y>
      <width>256</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>smtp.gmail.com</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>destinataire</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="paswd_4">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>210</y>
      <width>256</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="rcpt_4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>260</y>
      <width>256</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="uname_4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>160</y>
      <width>256</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1332</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
